test/e2e/scenario/recorder.scenario.ts

```diff
@@ -654,6 +670,236 @@ describe('recorder', () => {
         expect(segment.records.slice(3).every((record) => record.type !== RecordType.FullSnapshot)).toBe(true)
       })
   })
+
+  describe('layout viewport properties - ', () => {
+    createTest('getWindowWidth/Height reports using layout viewport dimensions')
+      .withRum({ enableExperimentalFeatures: ['visualviewport'] })
+      .withRumInit(initRumAndStartRecording)
+      .withSetup(bundleSetup)
+      .withBody(html`${VIEWPORT_META_TAGS}`)
+      .run(async ({ events }) => {
+        /**
+         * The purpose of this test is to check that browser dimensions are measuring the
+         * layout viewport (not visual viewport), and so they do not change when pinch zoom is applied.
+         *
+         * InnerWidth/Height on some devices/browsers are changed by pinch zoom.
+         * Read vendor discussions around the challenges of standardization:
+         * - https://bugs.chromium.org/p/chromium/issues/detail?id=489206
+         */
+        if (isGestureUnsupported()) {
+          return // No Fallback test
+        }
+        await resetViewport()
```

Comment:
what is the goal of that?
I've removed it and all tests seems to still be successful 🤔